jira/client.py

```diff
@@ -4381,6 +4381,26 @@ def delete_permissionscheme(self, id: str):
         self.permissionschemes.cache_clear()
         return data
 
+    def get_form(self, issue: str, formid: str) -> list[dict]:
```

Comment:
```suggestion
    def get_form(self, issue: str, form: str) -> list[dict]:
```

and change docstring.

Reply:
its an ID, not a form object, so i would not change that.

Reply:
My preference would be to use the generic term `form`, and use typehints to specify the types we allow, we then can handle a Form object in the future by simply extending the typehints, rather than adding a new argument to the function.

For example here, we might want to allow `str | int` and then in the function cast it to the string